src/test/java/org/openmrs/contrib/qaframework/page/ConditionsPage.java

```diff
@@ -25,7 +25,8 @@
 	private static final By TAB_ACTIVE = By.cssSelector("a[href='#ACTIVE']");
 	private static final By TAB_INACTIVE = By.cssSelector("a[href='#INACTIVE']");
 	private static final By FIRST_CONDITION_NAME = By.xpath("//table/tbody[2]/tr[1]/td[1]");
-	private static final By EDIT = By.cssSelector("i[title='Edit Condition: ']");
+	private static final By EDIT_ACTIVE_CONDITION = By.cssSelector("#ACTIVE tbody:nth-child(3) i.icon-pencil.edit-action.ng-scope");
+	private static final By EDIT_INACTIVE_CONDITION = By.cssSelector("#INACTIVE td:nth-child(4) i.icon-pencil.edit-action.ng-scope");
```

Comment:
```suggestion
private static final By EDIT_INACTIVE_CONDITION = By.cssSelector("#INACTIVE i.icon-pencil.edit-action.ng-scope");
```

Reply:
`#INACTIVE tbody:nth-child(4)  i.icon-pencil.edit-action.ng-scope` that is the element that i  get from https://qa-refapp.openmrs.org/openmrs so for the your suggestion, it throws 2 elements (2/2)